src/fragment-components/a-radiobutton.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { RadioButton, TextInput, Checkbox } from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+import { useFragment } from '../context';
+import { css } from 'emotion';
+import { getParentComponent, updatedState, Adder } from '../components';
+import image from './../assets/component-icons/radiobutton.svg';
+import { nameStringToVariableString,
+		angularClassNamesFromComponentObj,
+		reactClassNamesFromComponentObj } from '../utils/fragment-tools';
+
+export const ARadioButtonStyleUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<Checkbox
+			labelText='Disable button'
+			id='disable'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => setComponent({
+				...selectedComponent,
+				disabled: checked
+			})}/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent}/>
+	</>
+};
+
+
+export const ARadioButtonCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.labelText || ''}
+			labelText='Radio button label'
+			placeholder='Button value'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					labelText: event.currentTarget.value
+				});
+			}}
+		/>
```

Comment:
Since this changes the UI, this should be part of `ARadioButtonStyleUI` I would expect to have something like value here instead.